include/dieharder/libdieharder.h

```diff
@@ -274,9 +274,9 @@
  */
  typedef struct {
     FILE *fp;
-    off_t flen;
-    off_t rptr;
-    off_t rtot;
```

Comment:
There is no good reason for using `off_t`. No POSIX api is used, which requires `off_t`.